src/components/RadonOperator.vue

```diff
@@ -1,82 +1,75 @@
 <template>
   <div class="radon-operator">
-    <div class="selected-operator">
-      <el-select
-        data-test="select-btn"
-        class="operator-select"
-        v-model="selectedOptionValue"
-        @change="value => updateTemplateAndVariables(selectedOption.id, value)"
-        :clearable="true"
-        v-on:clear="
-          deleteOperator({
-            scriptId: scriptId,
-            operatorId: operator.id,
-          })
-        "
-      >
-        <el-option
-          v-for="(x, idx) in options"
-          :key="x.primaryText + selectedOption.id + idx"
-          :label="x.primaryText"
-          :value="x.value"
-        >
-          <span :data-test="`option-${idx}`" style="float: left">{{ x.primaryText }}</span>
-          <span style="float: right; color: #8492a6; font-size: 13px">
-            {{ x.secondaryText }}
-          </span>
-        </el-option>
-      </el-select>
-    </div>
-    <div class="with-arguments" v-if="hasArguments">
-      <div v-for="(argument, index) in selectedOperator.arguments" :key="argument.label + index">
-        <div v-if="argument.markupType === 'input'" class="input-container">
-          <el-input
-            class="input-operator"
-            size="small"
-            data-test="argument-input"
-            :placeholder="argument.label"
-            :value="argument.value ? argument.value.toString() : ''"
-            @input="value => updateTemplateAndVariables(argument.id, value, argument.type)"
-          />
-          <font-awesome-icon
-            data-test="variable-link-icon"
-            class="link"
-            v-show="hasVariables(argument.value)"
-            icon="link"
-          />
-        </div>
-        <div
-          :style="{ display: 'flex', flexDirection: 'column' }"
-          v-if="argument.markupType === 'select'"
-        >
-          <p>{{ argument.label }}</p>
-          <el-select
-            class="argument-options"
-            v-model="selectedArgumentValue[index]"
-            :options="argumentOptions[index]"
+    <div class="border">
+      <p class="label">Operator</p>
+      <Select
+        type="operator"
+        :value="selectedOption"
+        :options="operatorOptions"
+        @input="option => updateTemplateAndVariables(selectedOption.id, option.value)"
+      />
+      <p class="label" v-if="hasArguments">Arguments</p>
+      <div class="with-arguments" v-if="hasArguments">
+        <div v-for="(argument, index) in selectedOperator.arguments" :key="argument.label + index">
```

Comment:
Should we create a `EditorOperatorArgument` in order to reduce the complexity of this one?